rest-server/engines/src/main/scala/com/actionml/engines/ur/URDataset.scala

```diff
@@ -176,39 +176,38 @@ class URDataset(engineId: String, val store: Store) extends Dataset[UREvent](eng
     items.foreach(insertProperty)
   }
 
-  private def insertProperty(event: UREvent): Unit =
-    (for {
-      updateItem <- itemsDao.findOneByIdAsync(event.entityId).map(_.getOrElse {
-        logger.debug(s"No item found with id ${event.entityId}")
-        URItemProperties(event.entityId, Map.empty)
-      })
-      _ <- itemsDao.saveOneByIdAsync(
-        event.entityId,
-        URItemProperties(
-          _id = updateItem._id,
-          dateProps = updateItem.dateProps ++ event.dateProps,
-          categoricalProps = updateItem.categoricalProps ++ event.categoricalProps,
-          floatProps = updateItem.floatProps ++ event.floatProps,
-          booleanProps = updateItem.booleanProps ++ event.booleanProps
-        )
+  private def insertProperty(event: UREvent): Unit = {
```

Comment:
Note: this should ONLY be called for a name: $set reserved event name. There should be no reason to filter out other event names by the time we are in this function.

Reply:
yes, this method is called above only for $set items